docs/users/plugins-using-placeholderapi.md

```diff
@@ -215,6 +215,9 @@ If your plugin isn't shown here and you want it to be added, [read the Wiki READ
 - **[CitizensBooks](https://www.spigotmc.org/resources/37465/)**
     - [x] Supports placeholders.
     - [ ] Provides own placeholders. [Link]
+- **[ClaimChunk](https://www.spigotmc.org/resources/claimchunk.44458/)**
+    - [x] Supports placeholders.
+    - [x] Provides own placeholders. [**[Link](https://github.com/cjburkey01/ClaimChunk/wiki/Placeholder-API-Integration)**]
```

Comment:
Please make an entry in the placeholder-list.md file and link to that instead of an external site.

Reply:
Ah my bad, thank you!